### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (h4)

### ISSUE THREAD

**Issue by brunopulis** [brunopulis/brunopulis.com#187]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://dequeuniversity.com/rules/axe/4.0/color-contrast?application=msftAI))
#### Target application
[Home | Bruno Pulis](https://brunopulis.com/)
#### Element path
.mb-30px.col-md-6:nth-child(1) > .listfeaturedtag.h-100 > .row.h-100 > .col-lg-7.col-12.col-md-12 > .card-group.h-100 > .card > .card-body > h4
#### Snippet
`<h4 class="card-text">Confira tudo o que aconteceu no Minas Test Conference 2020, um artigo completo sobre as palestras e tudo que rolou no evento.</h4>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.24 (foreground color: #8f8f8f, background color: #ffffff, font size: 12.8pt (17.1px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 85.0.4183.83
This accessibility issue was found using Accessibility Insights for Web 2.21.1 (axe-core 4.0.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
